lambdas/select-cri/src/main/java/uk/gov/di/ipv/core/selectcri/SelectCriHandler.java

```diff
@@ -320,22 +322,25 @@ private Optional<JourneyResponse> getCriResponse(
     }
 
     private Optional<JourneyResponse> getMultipleDocCheckPage() {
-        if (drivingLicenceCriId.startsWith("stub")) {
+        CredentialIssuerConfig drivingLicenceConfig =
+                configService.getCredentialIssuerActiveConnectionConfig(drivingLicenceCriId);
+        if (drivingLicenceConfig.getActiveConnection() == "stub") {
```

Comment:
This wont work as activeConnection is a layer above, may need to use the original getCredentialIssuer method for this one